server/config.go

```diff
@@ -121,3 +181,57 @@ func (s *Service) ReplaceConfigSection(w http.ResponseWriter, r *http.Request) {
 
 	encodeJSON(w, http.StatusOK, res, s.Logger)
 }
+
+func validLogViewerUIConfig(cfg chronograf.LogViewerUIConfig) error {
+	if len(cfg.Columns) == 0 {
+		return fmt.Errorf("Invalid log viewer UI config: must have at least 1 column")
+	}
+
+	nameMatcher := map[string]bool{}
+	positionMatcher := map[int32]bool{}
+
+	for _, clm := range cfg.Columns {
```

Comment:
What's the idea here? The logic is a little hard to follow without context.

Reply:
This logic ensures that each column has a unique name, and that no two columns have the same position property.

I can write some comments for this portion of the code if it would help.